en/exercises/device_behaviour_check/summary.md

```diff
@@ -0,0 +1,4 @@
+
+The auditor checks staff devices for updated systems and software, anti-virus and other security capabilities, and identifies software running on computers and its current version.  The auditor checks for known vulnerabilities to any out of date software.
```

Comment:
I'm still not quite sure about calling it 'Device behaviour' assuming we are checking the status of a machine correct?

Reply:
am ok, leaving with old name and then merging the files which have changed. Let me know if I should merge and I will create another pull request with deleted 'Device behaviour' but merged file